temporalcli/commandsmd/commands.md

```diff
@@ -765,6 +765,7 @@ If there is no default Build ID, the result for the unversioned queue will be re
 * `--legacy-mode` (bool) - Enable a legacy mode for servers that do not support rules-based worker versioning. This mode only provides pollers info.
 * `--task-queue-type-legacy` (string-enum) - Task Queue type (legacy mode only). Options: workflow, activity. Default: workflow.
 * `--partitions-legacy` (int) - Query for all partitions up to this number (experimental+temporary feature) (legacy mode only). Default: 1.
+* `--disable-stats` (bool) - Disable task queue statistics.
 
```

Comment:
for reachability we use `report-reachability` as opposed to `disable-reachability` because it is expensive, and we wanted to avoid calls.
Is getting the stats also expensive? In that case we should use also use `report-stats`...

Reply:
Stats is not the cheapest thing (because it need partition fan-out) but still relatively cheap. I'm comfortable with having it enabled by default. wdyt @dnr ?

We may want to add caching for stats in the root partition in the future (if SDK starts to query it for poller scaling).